ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs

```diff
@@ -53,14 +55,17 @@ internal class BiggestGridDetectionPcuStrategy : AbstractGridDetectionStrategy {
         }
         private long CountProjectionPCU(MyCubeGrid grid) {
             long pcu = 0;
-            List<MyProjectorBase> projectors = new List<MyProjectorBase>();
-            var gridTerminalSystem = MyAPIGateway.TerminalActionsHelper.GetTerminalSystemForGrid(grid);
-            gridTerminalSystem.GetBlocksOfType(projectors);
-            foreach (var projector in projectors) {
-                List<MyObjectBuilder_CubeGrid> grids = projector.Clipboard.CopiedGrids;
-                foreach (MyObjectBuilder_CubeGrid objectBuilderCubeGrid in grids)
-                    pcu += objectBuilderCubeGrid.CubeBlocks.Count;
+             /*loop over the projectors in the grid */
+            foreach (var projector in grid.GetFatBlocks().OfType<MyProjectorBase>()) {
+                 /*if the projector is enabled and the entity id of the grid its on matches the one we are checking, count the projected blocks */
+                if (projector.CubeGrid.EntityId == grid.EntityId && projector.Enabled) {
```

Comment:
With the GridFatBlocks you no longer need that entity ID check

also I would like to not goo to far to the right. So you can just do 

if(!projector.Enabled)
continue;

instead.